README.md

```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
 > cargo build
 ```
 
-or to enable GTK 3.10 examples as well:
+or to enable GTK 3.1x, 3.2x, all examples as well:
 
 ```Shell
-> cargo build --features gtk_3_10
+> cargo build --features gtk_3_18
+> cargo build --features gtk_3_22_30
+> cargo build --all-features
```

Comment:
IMHO one version like 3.18 is enough.

Reply:
I agree with @EPashkin in here.

Reply:
Ok, I'd like to reword that so the reader understands which are the options available and how to find it. E.g. "check Cargo.toml [features] to see all specific GTK compile features available"

Reply:
To "enable them all", you can just enable the highest one. Also, in the cargo file, there is not only libraries versions handling but also features enablement. So I really like the first part of the sentence but way less the second one. ;)